starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -3,13 +3,64 @@
 """
 
 from __future__ import annotations
+from itertools import islice
+from queue import Empty
+from typing import Union, List
+import collections
+from starknet_devnet.blueprints.rpc.structures.types import (
+    BlockId,
+    Felt,
+    Address,
+)
+from starknet_devnet.blueprints.rpc.structures.responses import (
+    RpcEventsResult,
+)
 
-from typing import Union
-
-from starknet_devnet.blueprints.rpc.structures.payloads import Felt, Address
 from starknet_devnet.state import state
 
 
+def is_empty(value):
+    """
+    Return true if empty.
+    """
+    return not value and isinstance(value, collections.Iterable)
```

Comment:
I don't see the need for such an emptiness check. The most pythonic way is to use the object itself. Like, instead of doing `if is_empty(obj):` just doing `if obj`. What is the risk in not doing it that way?

Reply:
fixed